articles/applications/reference/wildcard-subdomains.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Wildcards for Subdomains
+description: Describes wildcards for subdomains function in application configuration.
+toc: true
+topics:
+  - applications
+contentType: reference
+useCase:
+  - build-an-app
+---
+# Wildcards for Subdomains
+
+You can use wildcards for subdomain URL registration in your application configuration in the Dashboard with these fields:
+
+* **Allowed Callback URLs**: Set of URLs to which Auth0 is allowed to redirect users after they authenticate.
+* **Allowed Logout URLs**: List of URLs to which you can redirect users after they log out from Auth0.
+* **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS).
+
+See [Application Settings Best Practices](/best-practices/application-settings) for recommended settings for applications, which suggests to avoid using wildcards for subdomains in application callbacks and allowed origins. If you choose to use them, you can use the star symbol (`*`) as a wildcard for subdomains, but it must be used in accordance with the following rules in order to properly function:
```

Comment:
Can we make this sentence pop out as a warning/notice? (yellow background or something of the sort)

> Avoid using wildcards for subdomains in application callbacks and allowed origins as it can make your application vulnerable to attacks. See [Application Settings Best Practices](/best-practices/application-settings) for this and other recommended settings.

And then continue the regular flow with

> To set up subdomain wildcards you can use the star symbol (`*`) as a wildcard for subdomains ...

WDYT?

Reply:
<img width="783" alt="Screen Shot 2020-02-21 at 11 55 01 AM" src="https://user-images.githubusercontent.com/1214600/75054425-09da7100-54a1-11ea-9076-2eb9f01b9fc7.png">